### PR TITLE
Add tests to multiple repos copy

### PR DESCRIPTION
**Pull request by kersommoura**
Add tests to verify new feature related to multiple repos copy. Test the
use of the new flag `additional_repos`.

https://pulp.plan.io/issues/5242
ref:#5242

### CODE REVIEW

**Comment by kersommoura** [comment]
Errata test scenario can be added based on the outcome of the https://pulp.plan.io/issues/5449

**Comment by kersommoura** [comment]
@dralley, if you have any comment.

File: pulp_2_tests/constants.py [hunk 1]
```diff
@@ -178,6 +178,7 @@
         'rpm_count': 2,
         'module_defaults_count': 2,
         'total_available_units': 7,
+        'packages': ['duck-0.7-1.noarch.rpm', 'kangaroo-0.3-1.noarch.rpm'],
```
Comment (kersommoura) [inline pulp_2_tests/constants.py]: This one was added to the enable errata tests later on.

File: pulp_2_tests/tests/rpm/api_v2/test_copy.py [hunk 2]
```diff
@@ -73,11 +77,14 @@ def test_all(self):
...
-            'criteria': {'filters': {}, 'type_ids': ['erratum']},
-        })
+        client.post(
+            urljoin(repos[1]['_href'], 'actions/associate/'),
+            {
+                'source_repo_id': repos[0]['id'],
+                'override_config': {'recursive': True},
+                'criteria': {'filters': {}, 'type_ids': ['erratum']},
+            },
+        )
```
Comment (dralley) [inline pulp_2_tests/tests/rpm/api_v2/test_copy.py]: These are all just formatting changes, right?

Comment (kersommoura) [reply pulp_2_tests/tests/rpm/api_v2/test_copy.py]: Yes.

Comment (kersommoura) [reply pulp_2_tests/tests/rpm/api_v2/test_copy.py]: I ran black against this module.

Comment (kersommoura) [reply pulp_2_tests/tests/rpm/api_v2/test_copy.py]: The class `MultipleSourceRepositoriesRecursiveCopyTestCase` is the one testing the newly added feature.

**Pull request merged**